cpp/test/Slice/errorDetection/WarningInvalidMetadata.err

```diff
@@ -1,21 +1,44 @@
+WarningInvalidMetadata.ice:12: warning: ignoring duplicate metadata: 'cpp:header-ext' has already been applied in this context
+WarningInvalidMetadata.ice:13: warning: missing required argument for 'cpp:header-ext' metadata
+WarningInvalidMetadata.ice:13: warning: ignoring duplicate metadata: 'cpp:header-ext' has already been applied in this context
+WarningInvalidMetadata.ice:14: warning: missing required argument for 'cpp:source-ext' metadata
+WarningInvalidMetadata.ice:14: warning: ignoring duplicate metadata: 'cpp:source-ext' has already been applied in this context
+WarningInvalidMetadata.ice:15: warning: missing required argument for 'cpp:source-ext' metadata
+WarningInvalidMetadata.ice:15: warning: ignoring duplicate metadata: 'cpp:source-ext' has already been applied in this context
+WarningInvalidMetadata.ice:16: warning: missing required argument for 'cpp:dll-export' metadata
+WarningInvalidMetadata.ice:16: warning: ignoring duplicate metadata: 'cpp:dll-export' has already been applied in this context
+WarningInvalidMetadata.ice:17: warning: missing required argument for 'cpp:dll-export' metadata
+WarningInvalidMetadata.ice:17: warning: ignoring duplicate metadata: 'cpp:dll-export' has already been applied in this context
+WarningInvalidMetadata.ice:20: warning: missing required argument for 'cpp:include' metadata
+WarningInvalidMetadata.ice:21: warning: missing required argument for 'cpp:include' metadata
+WarningInvalidMetadata.ice:24: warning: ignoring unknown metadata: 'unknown'
+WarningInvalidMetadata.ice:25: warning: ignoring unknown metadata: 'cpp:unknown'
+WarningInvalidMetadata.ice:26: warning: ignoring unknown metadata: 'bad:unknown'
+WarningInvalidMetadata.ice:31: warning: the 'cpp:no-default-include' metadata does not take any arguments
+WarningInvalidMetadata.ice:34: warning: 'amd' metadata cannot be specified as file metadata
+WarningInvalidMetadata.ice:40: warning: the 'cpp:header-ext' metadata only accepts one argument but a list was provided
+WarningInvalidMetadata.ice:40: warning: 'cpp:header-ext' metadata cannot be applied to sequences
+WarningInvalidMetadata.ice:44: warning: ignoring duplicate metadata: 'deprecated' has already been applied in this context
+WarningInvalidMetadata.ice:48: warning: ignoring unknown metadata: 'unknown'
+WarningInvalidMetadata.ice:48: warning: ignoring unknown metadata: 'cpp:unknown'
+WarningInvalidMetadata.ice:48: warning: ignoring unknown metadata: 'bad:unknown'
+WarningInvalidMetadata.ice:56: warning: the 'protected' metadata does not take any arguments
+WarningInvalidMetadata.ice:56: warning: the 'cpp:array' metadata does not take any arguments
+WarningInvalidMetadata.ice:62: warning: 'cpp:type' metadata cannot be applied operations with void return type
+WarningInvalidMetadata.ice:65: warning: 'cpp:array' metadata cannot be applied operations with void return type
+WarningInvalidMetadata.ice:68: warning: invalid argument 'my_string' supplied to 'cpp:type' metadata in this context.
```

Comment:
warning ends with period or not? Need to be consistent.

Reply:
They shouldn't end with periods, I inserted them by habit I guess.
Good catch. I removed this one, and there were other places where I also accidentally added them (also removed).